playground/frontend/lib/config/theme.dart

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/config/colors.dart';
+
+// TODO: add support for dart theme
+final theme = ThemeData(
+  brightness: Brightness.light,
+  primaryColor: LightColor.primary,
+  backgroundColor: LightColor.primaryBackground,
+  appBarTheme: const AppBarTheme(
+    color: LightColor.secondaryBackground,
+    elevation: 1,
+    centerTitle: false,
+  ),
+);
+
+class ThemeColors {
```

Comment:
This comment is not blocking of the PR.

Related to [this comment](https://github.com/apache/beam/pull/15626/files#r719676377), I wonder creating `final kLightTheme ...` and `final kDarkTheme` might be more readable instead of creating color classes.

Reply:
Thank you for your suggestion, could you please give an example for cases where we can't use `ThemeData`? For example, if we have a `Container` with some unique background color

Reply:
@ElessarST thank you for listening.  I have seen cases where the color assigned to a `TextStyle` property of a `Text` widget derived from `ThemeData`.  I imagine the same could be true of a `Container`.  I think the big picture is that as the application grows, changing styling becomes more difficult when hardcoded in specific places.

Reply:
@damondouglas yes, but I think `ThemeColors` will help with that because all colors will be here with switching between light and dark theme.